map/open-map-url/README.md

```diff
@@ -2,11 +2,11 @@
 
 Display a web map.
 
-![Image of open map URL](OpenMapURL.png)
+![](OpenMapURL.png)
```

Comment:
missing alt text here

Reply:
Yep these were removed before the sample viewer was fixed 😆